mods/ts/rules/gdi-structures.yaml

```diff
@@ -142,12 +142,12 @@ GAWEAP:
 	RevealsShroud:
 		Range: 4c0
 	RallyPoint:
-		Offset: 4,1
+		Offset: 3,1
```

Comment:
These were already set to the correct value in #10040.  We tested and showed that this does not fix #9678.